tests/mocha/json_test.js

```diff
@@ -0,0 +1,343 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+suite('JSON', function() {
```

Comment:
I know this is probably copied over from jsunit tests, but I think a more descriptive name would be better here. Possibly JSON Block Definitions?

Reply:
Done.